light_curves/code/gaia_functions.py

```diff
@@ -210,7 +204,7 @@ def Gaia_retrieve_median_photometry(coords_list , labels_list , object_names , g
             gaia_table = vstack( [gaia_table , gaia_search.get_data()[sel_min]] )
 
     if verbose > 0: print("\nSearch completed in {:.2f} seconds".format((time.time()-t1) ) )
-    if verbose > 0: print("Number of objects mached: {} out of {}.".format(len(gaia_table),len(object_names) ) )
+    if verbose > 0: print("Number of objects mached: {} out of {}.".format(len(gaia_table),len(coords_list) ) )
```

Comment:
```suggestion
    if verbose > 0: print("Number of objects matched: {} out of {}.".format(len(gaia_table),len(coords_list) ) )
```

I just noticed the spelling of "matched".

Reply:
fixed